hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionConfiguration.java

```diff
@@ -118,50 +95,45 @@
   private final int maxKeyValueSize;
 
   AsyncConnectionConfiguration(Configuration conf) {
+    ConnectionConfiguration connectionConf = new ConnectionConfiguration(conf);
```

Comment:
note this difference between master and branch-2 -- since we must maintain both an async and blocking configuration, I pulled this in here to DRY up the handling (and remove the duplicate logging)